helm/opendj/values.yaml

```diff
@@ -52,10 +52,9 @@ backup:
   enabled: false
   storageClass: "nfs"
   storageSize: "20Gi"
-  # Backup schedules for full and incrementals. This is a crontab format:
+  # Backup schedule. The first backup is a full, subsequent backups are incremenatal
```

Comment:
Add the line to indicate the schedule format is in crontab format.
# The schedule is in crontab format - minutes, hours, day-of-month, month, and day-of-week format